### PR TITLE
fix(zendesk): fixes issue where chat button doesnt show up when page …

### PR DESCRIPTION
**Pull request by blockdylanb**
…loads

## Description (optional)
Add a concise explanation of the changes.

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/components/SupportChat/components/SupportChatForGoldUserOnly/SupportChatForGoldUserOnly.tsx [hunk 1]
```diff
@@ -58,10 +58,15 @@ export const SupportChatForGoldUserOnly: SupportChatForGoldUserOnlyComponent = (
...
-      })
+      // FIXME: adding a 7 second delay because sendMessage was preiously firing before react was loaded
+      // in the iframe and the chat button wasn't showing up. The proper solution should wait for the iframe
+      // to tell wallet that it's loaded properly and will receive this message.
+      setTimeout(() => {
+        sendMessage('showChat', {
+          email: userData.email,
+          fullName: userFullname
+        })
+      }, 7000)
```
Comment (blockdylanb) [inline packages/blockchain-wallet-v4-frontend/src/components/SupportChat/components/SupportChatForGoldUserOnly/SupportChatForGoldUserOnly.tsx:69]: 7k ms was the smallest amount of time I tried that would make the button actually show up correctly in development

**Review by TheLeoB — APPROVED** [review]

**Comment by schnogz** [comment]
yeah this is gross and probably wont fix for all users.  why not set a listener in the useEffect for a ready event message passed from wallet helper?

**Comment by blockdylanb** [comment]
@schnogz yeah it's gross. Tried the other route but was running into weird issues and it wasn't working. This was supposed to be a quick fix, but got deprioritized. I'll just do the proper fix..

File: packages/blockchain-wallet-v4-frontend/src/components/SupportChat/components/SupportChatForGoldUserOnly/SupportChatForGoldUserOnly.tsx (hunk 1 above)
Comment (yasoo655) [reply packages/blockchain-wallet-v4-frontend/src/components/SupportChat/components/SupportChatForGoldUserOnly/SupportChatForGoldUserOnly.tsx:69]: My name is saad hussien i want to withdraw my money out of my payment mobile service Egypt Vodafone cash

**Pull request merged**